libexec/rc/network.subr

```diff
@@ -285,6 +285,8 @@ ifconfig_down()
 		else
 			/etc/rc.d/dhclient stop $1
 		fi
+	elif dhcpif $1; then
```

Comment:
This merge is incorrect. Its redundant with the dhcpif $1; then right above it. You'll want that section to look like this:

	if dhcpif $1; then
		if [ "$RC_SYSTEM" = "openrc" ] ; then
			sh /etc/rc.devd dhcpcd.${1} stop
		else
			/etc/rc.d/dhclient stop $1
		fi
		_cfg=0
	fi